distributed/comm/ucx.py

```diff
@@ -101,10 +101,15 @@ def device_array(n):
                 )
 
     pool_size_str = dask.config.get("rmm.pool-size")
+    enable_logging = dask.config.get("rmm.logging")
     if pool_size_str is not None:
         pool_size = parse_bytes(pool_size_str)
         rmm.reinitialize(
-            pool_allocator=True, managed_memory=False, initial_pool_size=pool_size
+            pool_allocator=True,
+            managed_memory=False,
+            initial_pool_size=pool_size,
+            logging=enable_logging,
+            log_file_name="",  # Something more useful here ;)
```

Comment:
We still need to fill this in with something. This is a bit tricky to get right as we would presumably want a different location depending on whether we are setting this on a worker, on the scheduler, or potentially on the client. However I'm not sure how we access that information from a context that is not inside one of those classes. Welcome thoughts on that though 🙂

Reply:
Maybe we should force the user to pass a path+filename prefix if logging is enabled and append the PID to it?